ui/src/components/FacetCard.js

```diff
@@ -60,8 +63,26 @@ class FacetCard extends Component {
         this.props.updateFacets(this.props.facet.name, facetValue.name, isInputChecked);
     }
 
-    isUnselected(facetValue) {
-        return this.state.selectedValues.length > 0 && this.state.selectedValues.indexOf(facetValue.name) < 0;
+    isDimmed(facetValue) {
+        if (this.state) {
+            return this.state.selectedValues.length > 0 && this.state.selectedValues.indexOf(facetValue.name) < 0;
+        }
+        return false;
+    }
+
+    /**
+     * @param facetValues FacetValue[] to sum counts over
+     * @param selectedValueNames Optional string[] to select a subset of facetValues to sum counts for
+     * @return count the total sum of all facetValue counts, optionally filtered by selectedValueNames
+     * */
+    sumFacetValueCounts(facetValues, selectedValueNames) {
+        let count = 0;
+        facetValues.forEach((value) => {
```

Comment:
I think this would be easier to read if:

if (selectedValueNames.length === 0)
   sum everything
else:
  ...

Reply:
Done